### ISSUE TITLE
Ready files from vendors table (FE)

### ISSUE THREAD

**Issue by plakitkelly** [keeleinstituut/tv-tolkevarav#722]
Project view - General tab - Ready files from vendors table
[Figma](https://www.figma.com/file/VqCwED3Dymn7L1gGoAOK27/EKI-t%C3%B5lkev%C3%A4rav?type=design&node-id=1999-80446&mode=design&t=LPKSeWgJnbQiDVWr-4)

1. File name in bold
2. "Tellijaga jagatud" wrap text in a table header

![image](https://github.com/keeleinstituut/tv-tolkevarav/assets/130659408/50a7a450-f4b3-4cee-b17e-e5578b0ef0be)
Currently the user can't see delete button, because "Tellijaga jagatud" is too wide and user has to scroll the table to click the delete button

**Comment by IngaSokman** [comment]
@plakitkelly The issue with the delete button being outside of the screen is that the file name is too long. How about making the file name more compact like this below? 

![Image](https://github.com/keeleinstituut/tv-tolkevarav/assets/130365953/b2dadeff-e675-447e-a541-d085204b1cce)

**Comment by plakitkelly** [comment]
@PlaksoBirgit can confirm this solution, but in figma is "Tellijaga jagatud" wrapped - it also gives more space for the name
![image](https://github.com/keeleinstituut/tv-tolkevarav/assets/130659408/46f9ac73-2455-4dc3-9d13-b9a87ecbfbb5)

**Comment by PlaksoBirgit** [comment]
Yes, it would be good to wrap also "Tellijaga jagatud" and if additionally needed also wrap the file name as well

**Comment by plakitkelly** [comment]
Tested in 15.04 - It's OK
